api/models/build.js

```diff
@@ -46,19 +46,24 @@ const sanitizeCompleteJobErrorMessage = message => message.replace(/\/\/(.*)@git
 
 const jobErrorMessage = (message = 'An unknown error occurred') => sanitizeCompleteJobErrorMessage(message);
 
-const jobStateUpdate = (buildStatus, build, completedAt) => {
-  let error = null;
+const jobStateUpdate = (buildStatus, build, timestamp) => {
```

Comment:
Looks good. Food for thought for future work: we have a good ol' fashioned state machine here and as the complexity increases we may want to consider making this a bit more explicit.

Reply:
agreed ... i'm not a fan of this function either .... could user refactoring